astropy/io/votable/converters.py

```diff
@@ -598,7 +598,7 @@ def __init__(self, field, base, arraysize, config=None, pos=None):
     def parse(self, value, config=None, pos=None):
         if config is None:
             config = {}
-        elif config['version_1_3_or_later'] and value == '':
+        elif 'version_1_3_or_later' in config and value == '':
```

Comment:
It looks like this change broke some tests.

Reply:
Yeah, I don't know why it is so confident that the config here would have `'version_1_3_or_later'`. Rather than diving into a rabbit hole, I modified the test to include this keyword instead. 🤷